saas/infra/prod/eu-central-1/app_plane/pooled/ec/terragrunt.hcl

```
@@ -0,0 +1,15 @@
+include "root" {
+  path = find_in_parent_folders()
+}
+
+include "env" {
+  path = "${get_terragrunt_dir()}/../../../../../_env/pooled/ec.hcl"
+}
+
+prevent_destroy = true
+
+inputs = {
+  deployment_template = "aws-cpu-optimized-v5"
```

in eu-central-1 we can use the arm version
```suggestion
  deployment_template = "aws-cpu-optimized-arm-v6"
```